etl/steps/data/garden/energy_institute/2023-06-26/statistical_review_of_world_energy.meta.yml

```diff
@@ -0,0 +1,709 @@
+        title: 'Wind Consumption - TWh'
+        unit: 'Terawatt-hours'
+        short_unit: 'TWh'
+        description: |
+          Figures are based on gross generation and do not account for cross-border electricity supply. "Input-equivalent" energy is the amount of fuel that would be required by thermal power stations to generate the reported electricity output.
+      wind_consumption_direct_twh:  # Added by OWID.
+        title: 'Wind Consumption (direct) - TWh'
+        unit: 'Terawatt-hours'
+        short_unit: 'TWh'
+        description: |
+          Figures are based on gross generation and do not account for cross-border electricity supply.
+      wind_electricity_generation_twh:
+        title: 'Wind Generation - TWh'
+        unit: 'Terawatt-hours'
+        short_unit: 'TWh'
+        description: |
+          Figures are based on gross generation and do not account for cross-border electricity supply.
+      cobalt_reserves_kt:
+        title: 'Cobalt - Reserves - kT'
+        unit: 'Thousand tonnes'
+        short_unit: 'kT'
+      coal_reserves_anthracite_and_bituminous_mt:
+        title: 'Coal - Reserves - Anthracite and bituminous'
+        unit: 'Million tonnes'
+        short_unit: 'MT'
+      coal_reserves_subbituminous_and_lignite_mt:
+        title: 'Coal - Reserves - Sub-bituminous and lignite'
+        unit: 'Million tonnes'
+        short_unit: 'MT'
+      coal_reserves_mt:
+        title: 'Coal - Reserves - Total'
+        unit: 'Million tonnes'
+        short_unit: 'MT'
+        description: |
+          Total proved reserves of coal are generally taken to be those quantities that geological and engineering information indicates with reasonable certainty can be recovered in the future from known reservoirs under existing economic and operating conditions. The data series for total proved coal reserves does not necessarily meet the definitions, guidelines and practices used for determining proved reserves at company level, for instance as published by the US Securities and Exchange Commission, nor does it necessarily represent BP's view of proved reserves by country. Reserves-to-production (R/P) ratio: If the reserves remaining at the end of any year are divided by the production in that year, the result is the length of time that those remaining reserves would last if production were to continue at that rate.
+      efficiency_factor:
+        title: Thermal equivalent efficiency factors
+        unit: ''
+        short_unit: ''
+        description: |
+          Thermal equivalent efficiency factors used to convert non-fossil electricity in input-equivalent to direct primary energy.
+
+  statistical_review_of_world_energy_fossil_fuel_prices:
+    variables:
+      # Coal prices.
+      coal_price_us_central_appalachian_spot_price_index_current_dollars_per_tonne:
+        title: Coal price - US Central Appalachian coal spot price index
+        unit: 'US dollars per tonne'
+        short_unit: 'US$'
```

Comment:
I think the short unit should just be '$'?

Reply:
It's in my to-do list to check the criteria on FASTT for these units, so I'll change it (and possibly variable names and chart titles) later.